### DISCUSSION TITLE
How to embed this in rustlang?

### DISCUSSION THREAD

**Discussion by sprappcom** [edubart/nelua-lang#265] [Q&A]
i would like to use nelua as a scripting language for webpage generation. looking for the fastest way possible.

is it possible to embed to rust and do a "hello world" demo?

**Comment by asumbek** [comment]
No you can't, Nelua is a language transpiled to C which is also compiled to native code. You probably meant embedding Lua or [Teal](https://github.com/teal-language/tl).
